code/modules/jobs/job_types/garrison/garrisonguard.dm

```diff
@@ -57,7 +57,7 @@
 	backr = /obj/item/weapon/shield/heater
 	backl = /obj/item/storage/backpack/satchel
 	beltr = /obj/item/weapon/sword/short
-	beltl = /obj/item/weapon/mace/cudgel
```

Comment:
they have a cudgel BECAUSE its weak, they are meant to aprehend people not kill them, its like a police baton. 

If you want to give them a mace its iffy, but keep the cudgel in some form.